react/components/ProductImages/components/Carousel/ThumbnailSwiper.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
-import { THUMB_SIZE, imageUrlForSize } from '../../../module/images'
+import { THUMB_SIZE } from '../../../module/images'
 import classNames from 'classnames'
 
-import styles from '../../styles.css'
 import { THUMBS_POSITION_HORIZONTAL } from '../../utils/enums'
 import { imageUrl } from '../../utils/aspectRatioUtil'
+import { useCssHandles } from 'vtex.css-handles'
```

Comment:
Our eslint should be reordering those imports, separating local and external dependency imports.